config/config.go

```diff
@@ -574,6 +576,9 @@ func (cfg *MempoolConfig) ValidateBasic() error {
 	if cfg.Size < 0 {
 		return errors.New("size can't be negative")
 	}
+	if cfg.MaxTxsTotalBytes < 0 {
```

Comment:
So setting either of these to 0 is allowed and will just cause us to always reject all txs?

Reply:
yes. I don't see why we'd want to forbid doing that